src/Logo.jsx

```diff
@@ -1,31 +1,18 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Logo = ({ src, alt, ...attributes }) => (
-  <img src={src} alt={alt} {...attributes} />
-);
-
-Logo.propTypes = {
-  src: PropTypes.string.isRequired,
-  alt: PropTypes.string.isRequired,
-};
-
-const LinkedLogo = ({
-  href,
-  src,
-  alt,
-  ...attributes
-}) => (
-  <a href={href} {...attributes}>
-    <img className="d-block" src={src} alt={alt} />
+const Logo = ({ content }) => (
+  <a href={content.href} className="logo">
+    <img className="d-block" src={content.src} alt={content.alt} />
   </a>
 );
 
-LinkedLogo.propTypes = {
-  href: PropTypes.string.isRequired,
-  src: PropTypes.string.isRequired,
-  alt: PropTypes.string.isRequired,
+Logo.propTypes = {
+  content: PropTypes.shape({
+    href: PropTypes.string.isRequired,
+    src: PropTypes.string.isRequired,
+    alt: PropTypes.string.isRequired,
+  })
 };
```

Comment:
I did this as a workaround since React allows editing children of a prop that is an object but not props directly. I think this could be undone if https://github.com/openedx/frontend-plugin-framework/pull/84 is merged.